eos/observable.cc

```diff
@@ -2229,7 +2245,7 @@ namespace eos
                     std::make_tuple("s_min", "s_max")),
 
 	    make_observable("Lambda_b->Lambdall::M_28@LowRecoil",
-                    &LambdaBToLambdaDilepton<LowRecoil>::integrated_m28,
+                    &LambdaBToLambdaDilepton<LowRecoil>::integrated_m8,
```

Comment:
Are you sure about this change? Seems like an accidental deleting ?!

Reply:
ah thanks for pointing this out to me. i don't remember touching this at all but it looks accidental as you said. i corrected this now. cheers